src/Neo/Ledger/TransactionRouter.cs

```diff
@@ -16,22 +16,19 @@
 
 namespace Neo.Ledger
 {
-    internal class TransactionRouter : UntypedActor
+    internal class TransactionRouter(NeoSystem system) : UntypedActor
```

Comment:
the core need to support netstandard2.1, does these change work?

Reply:
It's already used in other places, otherwise we should remove this rule